src/main/java/step2/model/LottoResults.java

```diff
@@ -0,0 +1,79 @@
+package step2.model;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public class LottoResults {
+
+    private static final int LOTTO_PRICE = 1000;
+
+    private Map<LottoResult, Long> lottoResults;
+
+    private LottoResults(Map<LottoResult, Long> result) {
+        this.lottoResults = Collections.unmodifiableMap(result);
+    }
+
+    public static LottoResults of(List<LottoResult> lottoResults) {
+        Map<LottoResult, Long> result = lottoResults.stream()
+            .collect(Collectors.groupingBy(Function.identity(), Collectors.counting()));
+        return new LottoResults(result);
+    }
+
+    public Long count(LottoResult lottoResult) {
+        return lottoResults.getOrDefault(lottoResult, 0L);
+    }
+
+    public Map<LottoResult, Long> getResult() {
+        Map<LottoResult, Long> result = getDefaultResult();
+        result.putAll(this.lottoResults);
+        return result;
+    }
+
+    public Double profit() {
```

Comment:
```suggestion
    public Double getProfit() {
```
메소드는 동사형으로 시작되어야 합니다!

Reply:
수정 완료!